clustered/client/src/main/java/org/ehcache/clustered/client/service/ClusteringService.java

```diff
@@ -59,8 +59,9 @@
    * Releases access to a {@link ServerStoreProxy} and the server-resident {@code ServerStore} it represents.
    *
    * @param serverStoreProxy a {@link ServerStoreProxy} obtained through {@link #getServerStoreProxy}
+   * @param isReconnect whether client is trying to reconnect
    */
-  void releaseServerStoreProxy(ServerStoreProxy serverStoreProxy);
+  void releaseServerStoreProxy(ServerStoreProxy serverStoreProxy, boolean isReconnect);
```

Comment:
Not a massive fan of this change... its bleeding a internal detail.  I'll swallow it for the sake of expediency though.